src/msg/async/AsyncMessenger.h

```diff
@@ -62,6 +63,23 @@ class Processor {
 	   entity_addrvec_t* bound_addrs);
   void start();
   void accept();
+  friend class AsyncMessenger;
+};
+
+class AsyncMessengerSocketHook : public AdminSocketHook {
+  AdminSocket& m_asok;
+  AsyncMessenger& m_msgr;
+  const std::string m_name;
+  const std::string m_command;
+
+ public:
+  AsyncMessengerSocketHook(
+      AdminSocket&, AsyncMessenger& m, const std::string& name);
+  ~AsyncMessengerSocketHook();
+  int call(
+      std::string_view command, const cmdmap_t& cmdmap, const bufferlist&,
+      Formatter* f, std::ostream& errss, ceph::buffer::list& out) override;
+  const std::string_view command() const { return m_command; };
```

Comment:
```suggestion
  std::string_view command() const { return m_command; };
```

`const` on `string_view` is redundant.